akamaiwaf.json

```diff
@@ -1561,42 +1562,92 @@
                 "type": "table"
             },
             "versions": "EQ(*)"
+        },
+        {
+            "action": "get siteshields",
+            "description": "Get akamai site shields ip ranges",
+            "type": "investigate",
+            "identifier": "get_siteshields",
+            "read_only": true,
+            "parameters": {},
+            "output": [
+                {
+                    "data_path": "action_result.status",
+                    "data_type": "string",
+                    "column_name": "Status",
+                    "column_order": 0,
+                    "example_values": [
+                        "success",
+                        "failed"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.cidr",
+                    "data_type": "string"
+                },
```

Comment:
Is it possible to include some example values for these output?

Reply:
Hello @siisar, Could you please address the suggested changes? Thank you!

Reply:
Hi, sorry fot the delay...
Yes, the values are cidr address, for instance  202.31.0.0/16, etc

Reply:
Added example value, as suggested.